pg-index-health/build.gradle

```diff
@@ -0,0 +1,44 @@
+dependencies {
+    api project(':pg-index-health-model')
+    api project(':pg-index-health-jdbc-connection')
+    implementation "org.slf4j:slf4j-api:${slf4jVersion}"
+
+    testImplementation project(':pg-index-health-testing')
+    testImplementation(testFixtures(project(":pg-index-health-model")))
+    testImplementation(testFixtures(project(":pg-index-health-jdbc-connection")))
+    testImplementation 'org.junit.jupiter:junit-jupiter-params'
+    testImplementation "ch.qos.logback:logback-classic:${logbackVersion}"
+    testImplementation "org.mockito:mockito-core:${mockitoVersion}"
+    testImplementation "org.awaitility:awaitility:${awaitilityVersion}"
+    testImplementation "org.apache.commons:commons-lang3:${commonsLang3Version}"
+
+    //testImplementation "org.apache.commons:commons-dbcp2:${dbcp2Version}"
```

Comment:
If it's done intentionally, it's useful to share this knowledge with others.

Reply:
Fixed